packages/web-components/src/components/masthead/masthead-l1.ts

```diff
@@ -656,7 +730,7 @@ class C4DMastheadL1 extends StableSelectorMixin(LitElement) {
             : ''}
           ${footer
             ? html`
-                <a
+                <a part="l1-dropdown-viewall"
```

Comment:
This is good. Found another missing `part` on line 946. The `<a>` around an L1 column heading. I would suggest `part="l1-dropdown-item-link l1-dropdown-item-link--heading"`